doc-src/SASS_CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 * Add a `$weight` parameter to `invert()`.
 
+* Add support for nested maps with `map-get()` and `map-has-key()` by passing more than 2 arguments
```

Comment:
Please limit Markdown lines to 80 characters and make sure they're full sentences with proper punctuation.

In general, please try to match the surrounding style in prose or code.